### DISCUSSION TITLE
How can we use two different reporter plugins with Cypress?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#15507] [Questions and Help]
Hi, can you please provide suggestions on how we can have these two reporters in our framework? Thanks.

https://github.com/Vivify-Ideas/cypress-testrail-reporter

https://github.com/LironEr/cypress-mochawesome-reporter

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/2043287/111237864-2ddc8900-85b3-11eb-8e37-65acd7ec6f57.png">

**Comment by bahmutov** [comment]
See our section on multiple reporters https://on.cypress.io/reporters#Multiple-reporters
